src/property.html

```diff
@@ -0,0 +1,1066 @@
+                    x="25"
+                    y="32"
+                    width="15"
+                    height="4"
+                    rx="2"
+                    fill="white"
+                  />
+                </svg>
+              </div>
+            </section>
+          </div>
+          <!-- section first -->
+          <section
+            class="border-bottom px-3 py-12 sm:px-8 lg:py-32 2xl:py-52 lg:pl-10 lg:pr-64 2xl:px-20 bg-linear-gradient flex flex-col gap-3 2xl:gap-4"
+          >
+            <div
+              class="font-semibold text-xl sm:text-4xl 2xl:text-5xl text-color"
+            >
+              Find Your Dream Property
+            </div>
+            <div
+              class="font-medium text-sm sm:text-base 2xl:text-lg text-color-addition-info"
+            >
+              Welcome to Estatein, where your dream property awaits in every
+              corner of our beautiful world. Explore our curated selection of
+              properties, each offering a unique story and a chance to redefine
+              your life. With categories to suit every dreamer, your journey
+            </div>
+          </section>
+        </div>
+        <!-- section second search bars -->
+        <section
+          class="w-full px-3 sm:px-10 2xl:px-40 lg:absolute -bottom-32 flex flex-col gap-5 lg:gap-0 lg:items-center"
+        >
+          <!-- search bar solo div -->
+          <div
+            class="w-full lg:w-4/5 col-span-3 border-button border-outline-two p-2.5 sm:p-4 2xl:p-5 flex items-center justify-between rounded-xl lg:rounded-b-none px-3.5 py-5 gap-5 bg-color-all"
+          >
+            <p
+              class="text-color-addition-info font-medium text-base sm:text-xl 2xl:text-2xl"
+            >
+              Search For A Property
+            </p>
+            <a
+              href="#"
+              class="btn-bg-color py-3.5 px-5 rounded-lg sm:flex items-center gap-1.5 2xl:gap-2 text-sm 2xl:text-lg"
+              ><img
+                src="/public/assets/propertypage/search.svg"
+                alt="search-icon"
+              />
+              <p class="text-color hidden sm:inline-block">Find Property</p>
+            </a>
+          </div>
+          <!-- 4 bars -->
+          <div
+            class="w-full bg-for-logo-section rounded-xl flex flex-col lg:grid grid-cols-5 gap-5 p-2.5"
+          >
+            <!-- first -->
+            <div
+              class="flex justify-between gap-5 2xl:gap-12 items-center rounded-lg py-3 px-3.5 2xl:p-5 border-button text-color bg-color-all"
+            >
+              <div class="grid grid-flow-col items-center gap-2">
+                <div>
+                  <img
+                    src="/public/assets/propertypage/location.svg"
+                    alt="location"
+                  />
+                </div>
+                <div class="border-left h-5"></div>
+                <div class="text-color-addition-info text-sm 2xl:text-lg">
```

Comment:
shitty class name text-color-addtional-info, change it everywhere in the project please

Reply:
changed